data/default_imsim_configs

```diff
@@ -1,5 +1,9 @@
 [electronics_readout]
 readout_time = 3.
+dark_current = 0.02
+bias_level = 1000.
+pcti = 1e-6
```

Comment:
Maybe a inline comment `# Parallel CTI level` for here and serial for non-experts who might not be able to figure out the acronym?